examples/simple/stdout_exporter.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <iostream>
+
+namespace trace    = opentelemetry::trace;
+namespace nostd    = opentelemetry::nostd;
+namespace sdktrace = opentelemetry::sdk::trace;
+
+class StdoutExporter final : public sdktrace::SpanExporter
```

Comment:
Probably not to be resolved in this PR but: should we have a canonical StdoutExporter in the SDK?

Reply:
Yes, I think so. But that should go in a separate PR (than the `StdoutExporter` here can be replaced).

I'll write an issue for that. That could be a good issue for an intern to work on.